.github/workflows/ci.yml

```diff
@@ -35,9 +35,33 @@ jobs:
         with:
           version: "${{ matrix.NXF_VER }}"
 
+      - name: Cache test data
```

Comment:
I could swear there was a way where we don't have to repeat this big block of code for every test?

Reply:
Probably, but we'll rewrote the tests soon enough, so we'll fix that in a future PR